src/components/Main/MovieItem/MovieItem.tsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import './MovieItem.css';
 
-const MovieItem = ({ image, name, year, genre }) => (
+interface MovieItemProps { image: string, name: string, year: string, genre: string }
```

Comment:
We have two similar interfaces w/ `MovieItem` definition. Can we reuse one of them?